README.md

```diff
@@ -61,140 +64,45 @@ headers, body = event.to_http()
-    {
-        "ce-specversion": "1.0",
-        "content-type": "application/octet-stream",
-        "ce-type": "word.found.name",
-        "ce-id": "96fb5f0b-001e-0108-6dfe-da6e2806f124",
-        "ce-time": "2018-10-23T12:28:22.4579346Z",
-        "ce-source": "<source-url>",
-    },
-    b"this is where your CloudEvent data is, including image/jpeg",
-    lambda x: x,  # Do not decode body as JSON
-)
-```
+#### Request to CloudEvent
 
-Creating HTTP request from CloudEvent:
+The code below shows how to consume a cloudevent using the popular python web framework
+[flask](https://flask.palletsprojects.com/en/1.1.x/quickstart/):
 
 ```python
-from cloudevents.sdk import converters
-from cloudevents.sdk import marshaller
-from cloudevents.sdk.converters import structured
-from cloudevents.sdk.event import v1
-
-event = (
-    v1.Event()
-    .SetContentType("application/json")
-    .SetData('{"name":"john"}')
-    .SetEventID("my-id")
-    .SetSource("from-galaxy-far-far-away")
-    .SetEventTime("tomorrow")
-    .SetEventType("cloudevent.greet.you")
-)
-
-m = marshaller.NewDefaultHTTPMarshaller()
-
-headers, body = m.ToRequest(event, converters.TypeStructured, lambda x: x)
-```
-
-## HOWTOs with various Python HTTP frameworks
-
-In this topic you'd find various example how to integrate an SDK with various HTTP frameworks.
+from flask import Flask, request
 
-### Python requests
-
-One of popular framework is [`requests`](http://docs.python-requests.org/en/master/).
-
-#### CloudEvent to request
-
-The code below shows how integrate both libraries in order to convert a CloudEvent into an HTTP request:
-
-```python
 from cloudevents.sdk.http_events import CloudEvent
-from cloudevents.sdk.types import converters
-
-def run_binary(event: CloudEvent, url):
-    # If event.data is a custom type, set data_marshaller as well
-    headers, data = event.to_http(converters.TypeBinary)
 
-    print("binary CloudEvent")
-    for k, v in headers.items():
-        print("{0}: {1}\r\n".format(k, v))
-    print(data)
-    response = requests.post(url,
-                             headers=headers,
-                             data=data)
-    response.raise_for_status()
+app = Flask(__name__)
 
 
-def run_structured(event: CloudEvent, url):
-    # If event.data is a custom type, set data_marshaller as well
-    headers, data = event.to_http(converters.TypeStructured)
+# create an endpoint at http://localhost:/3000/
+@app.route("/", methods=["POST"])
+def home():
+    # create a CloudEvent
+    event = CloudEvent.from_http(request.get_data(), request.headers)
 
-    print("structured CloudEvent")
-    print(data)
+    # you can access cloudevent fields as seen below
+    assert event['specversion'] == "1.0"
+    print(f"Found CloudEvent from {event['source']}")
 
-    response = requests.post(url,
-                             headers=headers,
-                             data=data)
-    response.raise_for_status()
+    if event['type'] == 'com.readme.binary':
```

Comment:
Noted above, we should change the type to not have the words `structured` or `binary`.

This could confuse folks trying to understand if a event is sent over one of these message modes.